packages/assets/scss/01-atoms/_table.scss

```diff
@@ -5,6 +5,26 @@
   border-collapse: collapse;
   width: 100%;
 
+  .ma__table__caption,
+  & td > table caption {
+    color: $c-font-heading;
+    font-weight: $fonts-bold;
+    line-height: 1.3;
+    text-align: left;
+    margin-bottom: 0.8em;
+
+    &.hide {
+      display: none;
+    }
+
+    &__content {
+      display: inline-block;
+      width: 100%;
+    }
+
+    // TODO:  ADD STYLE FOR ONE COL LAYOUT

```

Comment:
what do you mean by one col layout?

Reply:
This is what I forgot to remove. I initially misunderstood the task as implementing the Mayflower in openmass.  It was a note for myself.  It's removed.